src/app/ApplicationTemplate.Mobile/iOS/Info.plist

```diff
@@ -59,8 +59,8 @@
 		</array>
 		<!--
 		Adjust this to your application's encryption usage.
```

Comment:
I think we should add comments to explain this, some apps should not have this property already at false, there are legal consequence to this.

Reply:
Oh it's only for my sample app, it won't be like this in main.

Reply:
It could be a nice suggestion for the main branch.